content/blog/use-redfish-and-ipv6-to-find-lost-servers.md

```diff
@@ -0,0 +1,473 @@
+network port using the IPv6 link local address and zone index.
+
+```shell
+curl --noproxy \* --insecure --silent --location \
+     --user username:password                    \
+     https://[FE80::9618:82FF:FE71:A07B%eno1]/redfish/v1/Managers/1/EthernetInterfaces/1 | \
+     jq '.IPv4Addresses[]'
+{
+  "Address": "192.168.1.45",
+  "AddressOrigin": "Static",
+  "Gateway": "192.168.1.1",
+  "SubnetMask": "255.255.252.0"
+}
+```
+
+```shell
+ilorest login https://[FE80::9618:82FF:FE71:A07B%eno1] -u <username> -p password
+ilorest select EthernetInterface.
+ilorest get Ipv4Addresses --filter Name="Manager Dedicated*" --json
+{
+  "IPv4Addresses": [
+    {
+      "Address": "192.168.1.45",
+      "AddressOrigin": "Static",
+      "Gateway": "192.168.1.1",
+      "SubnetMask": "255.255.252.0"
+    }
+  ]
+}
+
+ilorest logout
+```
+
+```shell
+# Use of PowerShell core (7) and Redfish basic authentication
+$Credentials = "username:password"
+$base64EncodedCreds = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($Credentials))
+$Headers = @{}
+$Headers["Content-Type"] = "application/json"
+$Headers["Authorization"] = "Basic $base64EncodedCreds"
+
+$Ipv6Address="FE80::9618:82FF:FE71:A07B%22"
+$Uri="https://[$Ipv6Address]/redfish/v1/Managers/1/EthernetInterfaces/1"
+
+$response = Invoke-WebRequest -Uri $Uri -Method GET -Headers $Headers -SkipCertificateCheck
+$JsonResponse = $response.Content | ConvertFrom-Json
+$JsonResponse.IPv4Addresses
+Address      AddressOrigin Gateway     SubnetMask
+-------      ------------- -------     ----------
+192.168.1.47 Static        192.168.1.1 255.255.252.0
+```
+
+### Multicast addresses
+
+An important concept in IPv6 is the absence of broadcast addresses. Instead, IPv6 uses multicast addresses. The exhaustive list of the pre-defined multicast addresses is present in <a href="https://datatracker.ietf.org/doc/html/rfc4291#section-2.7.1" target="_blank">RFC 4291</a>. Among them is the all nodes `FF02::1` address.
+
+From you brick-saver, when you issue a `ping FF02::1%Index` command, each and every IPv6 interfaces present on the LAN must reply with their link local address.
+By doing so, you can build the table of all your IPv6 neighbors. You will find more detail on how to use this table in the second scenario below.
+
+## Recovery process
+
+The recovery process presented in this blog post relies entirely on EUI-64 IPv6 link local addresses, and thus on MAC addresses. If you don't have
+the list of the affected iLO MAC addresses, you need to go through the next paragraph.
+
+If you have that list and your systems are iLO 5 based or later with
+<a href="https://servermanagementportal.ext.hpe.com/docs/redfishservices/ilos/ilo6/ilo6_161/ilo6_network_resourcedefns161/#oemhpeipv6"
+target="_blank">RFC7217</a>
+disabled, you can skip the next paragraph and study the second scenario.
```

Comment:
```suggestion
If you already have that list and your systems are iLO 5 based or later, with
<a href="https://servermanagementportal.ext.hpe.com/docs/redfishservices/ilos/ilo6/ilo6_161/ilo6_network_resourcedefns161/#oemhpeipv6"
target="_blank">Oem.Hpe.IPv6.RFC7217Enabled</a>
turned off, you can skip the next paragraph and study the second scenario.
```

To avoid "or" confusion.  
Also I'm not able to find reference to this resource in iLO 5 doc. according to your link it says beeing introduced in   iLO6 1.61 .

Reply:
Granted, there is a confusion here, because RFC 7217 is not implemented in iLO 5 and will never be implemented. I completely rephrased this sentence to avoid this confusion.